src/windows.rs

```diff
@@ -9,19 +8,30 @@ use std::os::windows::io::{AsRawHandle, FromRawHandle};
 use std::path::Path;
 use std::ptr;
 
+use self::winapi::shared::minwindef::{BOOL, DWORD};
+use self::winapi::shared::winerror::ERROR_LOCK_VIOLATION;
+use self::winapi::um::winbase::GetFileInformationByHandleEx;
+use self::winapi::um::fileapi::{FILE_ALLOCATION_INFO, FILE_STANDARD_INFO, GetDiskFreeSpaceW};
+use self::winapi::um::fileapi::{GetVolumePathNameW, LockFileEx, UnlockFile, SetFileInformationByHandle};
```

Comment:
I think I'd prefer if you added

```rust
#[cfg(windows)]
extern crate winapi;
```

to `lib.rs` so that you don't need these `self::` prefixes, I find those confusing since they imply that the symbols are coming from the current crate.